### ISSUE TITLE
Summary endpoint is broken due to lingering `fileCount` aggregation

### ISSUE THREAD

**Issue by noah-aviel-dove** [DataBiosphere/azul#3469]
https://github.com/DataBiosphere/azul/pull/3352#issuecomment-929012848

**Comment by noah-aviel-dove** [comment]
```diff 
Index: src/azul/service/index_query_service.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/azul/service/index_query_service.py b/src/azul/service/index_query_service.py
--- a/src/azul/service/index_query_service.py	(revision 22175e37504dc6d51d9d5329105d838bf030c8f3)
+++ b/src/azul/service/index_query_service.py	(date 1632821356153)
@@ -148,7 +148,6 @@
             'files': [
                 'totalFileSize',
                 'fileFormat',
-                'fileCount'
             ],
             'samples': [
                 'organTypes',
Index: src/azul/service/elasticsearch_service.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/azul/service/elasticsearch_service.py b/src/azul/service/elasticsearch_service.py
--- a/src/azul/service/elasticsearch_service.py	(revision 22175e37504dc6d51d9d5329105d838bf030c8f3)
+++ b/src/azul/service/elasticsearch_service.py	(date 1632821356145)
@@ -538,9 +538,6 @@
             assert entity_type == 'projects', entity_type
 
         cardinality_aggregations = {
-            'files': {
-                'fileCount': 'contents.files.uuid'
-            },
             'samples': {
                 'specimenCount': 'contents.specimens.document_id',
                 'speciesCount': 'contents.donors.genus_species',
```

**Comment by hannes-ucsc** [comment]
This is a regression from PR #3352 for #3318. That fact was obvious from the moment the `develop` build failed for that PR. For that reason, I don't think we should create another issue but instead file a follow-up PR for #3318 from a branch that includes the fix for this and all changes from PR #3352.

**Comment by hannes-ucsc** [comment]
The develop branch was reset to undo the changes from PR #3352 for #3318.
